core/iwasm/compilation/aot_llvm.h

```diff
@@ -291,18 +281,14 @@ typedef struct AOTCompContext {
     uint64 flags[8];
 
     /* LLVM execution engine required by JIT */
-#if WASM_ENABLE_LAZY_JIT != 0
-    LLVMOrcLLJITRef orc_lazyjit;
+    LLVMOrcLLJITRef orc_jit;
```

Comment:
Seems that we don't use the `WAMR_BUILD_JIT` macro to control the fields and the code in `core/iwasm/compilation`?